pkg/projects/resource_project_user.go

```diff
@@ -0,0 +1,214 @@
+package projects
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type User struct {
+	Name  string   `json:"name"`
+	Roles []string `json:"roles"`
+}
+
+type Comparable interface {
```

Comment:
lol! #javarules

Reply:
Interesting, I took this cue from C# which has `IComparable` with these two property/function. I will update the struct name.

Reply:
LOL my memory tricked me. `IComparable` has `CompareTo()`, similar to java